core/src/main/scala/model/Metric.scala

```diff
@@ -18,6 +18,46 @@ package freestyle
 package opscenter
 package model
 
-case class Metric[T](metric: String, microservice: String, node: String, value: T, date: Long) {
-  override def toString(): String = s"$microservice.$node $date $metric $value"
+import freestyle.opscenter.protobuf.metric.{ListMetricsProto, MetricProto}
+
+case class Metric(metric: String, microservice: String, node: String, value: Float, date: Long) {
+  def toProto: MetricProto = MetricProto(
+    metric = metric,
+    microservice = microservice,
+    node = node,
+    value = value,
+    date = date
+  )
+}
+
+object Metric {
+
+  def apply(proto: MetricProto): Metric = Metric(
+    metric = proto.metric,
+    microservice = proto.microservice,
+    node = proto.node,
+    value = proto.value,
+    date = proto.date
+  )
+
+  def apply(bytes: Array[Byte]): Metric =
+    Metric(MetricProto.parseFrom(bytes))
+
+}
+
+case class ListMetrics(listMetrics: Seq[MetricProto]) {
```

Comment:
Let's use `List` instead of `Seq`

Reply:
Totally agree. Done ...

Reply:
I put a Seq because it was the type that was returning Proto when I set the field as repetead. I  just changed the custom collection type and that repeated field `metricsList` return a List right now. `[(scalapb.field).collection_type="List"];`